data-raw/remaining_carbon_budgets.R

```diff
@@ -1,15 +1,55 @@
 # styler: off
-remaining_carbon_budgets <- tibble::tribble(
-  ~scenario_source,  ~ald_sector, ~remaining_carbon_budget,           ~unit, ~weighting_factor,
-        "GECO2021",       "coal",                 84934.79, "Mt CO2 (2019)",              0.25,
-        "GECO2021",        "oil",                 82987.79, "Mt CO2 (2019)",              0.24,
-        "GECO2021",        "gas",                 49094.21, "Mt CO2 (2019)",              0.14,
-        "GECO2021",      "power",                 91121.17, "Mt CO2 (2019)",              0.27,
-        "GECO2021",      "steel",                 10746.03, "Mt CO2 (2019)",              0.03,
-        "GECO2021", "automotive",                  11338.9, "Mt CO2 (2019)",              0.03,
-        "GECO2021",        "hdv",                  7388.01, "Mt CO2 (2019)",              0.02,
-        "GECO2021",   "aviation",                  5963.32, "Mt CO2 (2019)",              0.02
+# The Extended file data for WEO 2023 is stored here:
+# https://portal.azure.com/#view/Microsoft_Azure_FileStorage/FileShareMenuBlade/~/browse/storageAccountId/%2Fsubscriptions%2Ffeef729b-4584-44af-a0f9-4827075512f9%2FresourceGroups%2FRMI-SP-PACTA-PROD%2Fproviders%2FMicrosoft.Storage%2FstorageAccounts%2Fpactarawdata/path/scenario-sources/protocol/SMB
+# The specific file is 'weo_2023-20240222/WEO2023 extended data/WEO2023_Extended_Data.xlsx'
+# Please download the file and replace the dummy path in variable 'file_scenario_emissions_data'
+# to the path on your computer before running the code.
+file_scenario_emissions_data <- "PATH/TO/WEO2023/SCENARIO/EXTENDED/DATA"
+carbon_emissions <- read_xlsx(file_scenario_emissions_data,
+                              sheet = "World CO2 Emissions",
+                              range = "J7:AP45") %>%
+  select(-c("2022...5", "2030...6", "2035...7", "2040...8", "2045...9", "2050...10", "...11", "2030...12", "2050...13",
+            "2022...15", "2030...16", "2035...17", "2040...18", "2045...19", "2050...20", "...21", "2030...22", "2050...23", "...24",
+            "...31", "2030...32", "2050...33", "...14")) %>%
+  rename(
+    sector = "...1",
+    emissions_2022 = "2022...25",
+    emissions_2030 = "2030...26",
+    emissions_2035 = "2035...27",
+    emissions_2040 = "2040...28",
+    emissions_2045 = "2045...29",
+    emissions_2050 = "2050...30",
+  ) %>%
+  filter(sector %in% c("Coal", "Oil", "Natural gas", "Electricity and heat sectors", "Iron and steel**", "Cement**", "Passenger cars", "Aviation")) %>%
+  mutate(
+    scenario_source = "WEO2023",
+    unit = "Mt CO2 (2022)"
   )
+
+carbon_emissions <- carbon_emissions[-5,]
+carbon_emissions <- carbon_emissions[-5,]
+carbon_emissions <- carbon_emissions[-5,]
+carbon_emissions <- carbon_emissions[-5,]
+carbon_emissions <- carbon_emissions[-5,]
+carbon_emissions <- carbon_emissions[-5,]
```

Comment:
There's probably a more readable/ interpretable way to do this?

Reply:
<img width="748" alt="image" src="https://github.com/RMI-PACTA/pacta.executive.summary/assets/73108425/a5a172a3-ff1f-49b3-8f77-fcb06496fdd6">
The issue is that those lines get the same labels

Reply:
I fixed this